### ISSUE TITLE
Weird bug when doing syntax checks in rakudo-star derived Dockerfile

### ISSUE THREAD

**Issue by japhb** [Raku/docker#44]
See https://gist.github.com/japhb/96ab63db5f19afb29aae3f5b92b1b834 for details.

Essentially if `raku -c` is run under `USER raku`, and it is checking a module with an exported MAIN, it will actually run MAIN *in addition* to syntax checking it.  If the `USER raku` is commented out, `raku -c` only does the syntax check as requested.

I tested it with `rakudo-star:2020.10`; happy to retest with latest rakudo-star when https://github.com/Raku/docker/issues/43 is completed.

**Comment by jmaslak** [comment]
I don't think this is a Docker image bug so I'm going to close this bug. I referenced above a bug I filed against rakudo for this issue.

The issue is that the Dockertest.rakumod file is in a directory that the raku user does not have write access into, and for some reason this makes Raku actually run the file.  The work around would be to set the privileges on /app so it is writable by the raku user.
